webpack.config.ts

```diff
@@ -68,6 +68,7 @@ export default function (cli_env: Record<string, boolean> = {}, argv: any) {
                 { test: /(async-call|webextension).+\.js$/, enforce: 'pre', use: ['source-map-loader'] },
                 { parser: { requireEnsure: false, amd: false, system: false } },
                 getTypeScriptLoader(),
+                getURLLoader(),
```

Comment:
Note: If we want to use this kind of loader, we need to re-implement our SSR code to support it.

Reply:
And I also suggest not introduce this because, in Webpack 5, it is able to use `new URL(import.meta.url, "./x.png")` to do the same thing but more grace.

Reply:
Removed.